src/aap_eda/core/management/commands/create_initial_data.py

```diff
@@ -479,17 +505,26 @@ def _copy_scm_credentials(self):
                 "username": cred.username,
                 "password": cred.secret.get_secret_value(),
             }
-            eda_cred = models.EdaCredential.objects.create(
+            eda_cred, created = models.EdaCredential.objects.get_or_create(
                 name=cred.name,
-                description=cred.description,
-                managed=False,
-                credential_type=cred_type,
-                inputs=inputs_to_store(inputs),
-            )
-            models.Project.objects.filter(credential=cred).update(
-                eda_credential=eda_cred, credential=None
+                defaults={
+                    "description": cred.description,
+                    "managed": False,
+                    "credential_type": cred_type,
+                    "inputs": inputs_to_store(inputs),
+                },
             )
-            cred.delete()
+            if created:
+                models.Project.objects.filter(credential=cred).update(
+                    eda_credential=eda_cred, credential=None
+                )
+                cred.delete()
+            else:
+                warn_msg = (
+                    f"Git Credential {cred.name} already converted to "
+                    "EdaCredential. Skip the duplicated one."
+                )
+                LOGGER.warning(warn_msg)
```

Comment:
Keep them in sync, should we change this into the `info` level too?

Reply:
sorry, I missed this.